### PR TITLE
Update list of enforced lints

### PR DESCRIPTION
**Pull request by nshahan**
- Bump version number.
- Update README with considered but unused lints.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,31 @@
...
+
+- Enforce 8 new lint rules:
+  - [`avoid_single_cascade_in_expression_statements`]
+  - [`await_only_futures`]
+  - [`prefer_inlined_adds`]
+  - [`sort_child_properties_last`]
+  - [`unnecessary_brace_in_string_interps`]
+  - [`unnecessary_getters_setters`]
+  - [`use_full_hex_values_for_flutter_colors`]
+  - [`unsafe_html`]
```
Comment (pq) [inline CHANGELOG.md:11]: consider sorting `unsafe_html` before `use_full_hex_values_for_flutter_colors`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,31 @@
...
+[`avoid_single_cascade_in_expression_statements`]: https://dart-lang.github.io/linter/lints/avoid_single_cascade_in_expression_statements.html
+[`await_only_futures`]: https://dart-lang.github.io/linter/lints/await_only_futures.html
+[`prefer_inlined_adds`]: https://dart-lang.github.io/linter/lints/prefer_inlined_adds.html
+[`sort_child_properties_last`]: https://dart-lang.github.io/linter/lints/sort_child_properties_last.html
+[`unnecessary_brace_in_string_interps`]: https://dart-lang.github.io/linter/lints/unnecessary_brace_in_string_interps.html
+[`unnecessary_getters_setters`]: https://dart-lang.github.io/linter/lints/unnecessary_getters_setters.html
+[`use_full_hex_values_for_flutter_colors`]: https://dart-lang.github.io/linter/lints/use_full_hex_values_for_flutter_colors.html
+[`unsafe_html`]: https://dart-lang.github.io/linter/lints/unsafe_html.html
+[`avoid_empty_else`]: https://dart-lang.github.io/linter/lints/avoid_empty_else.html
+[`curly_braces_in_control_structures`]: https://dart-lang.github.io/linter/lints/curly_braces_in_flow_control_structures.html
```
Comment (pq) [inline CHANGELOG.md:27]: Consider sorting?

File: README.md [hunk 3]
```diff
@@ -143,7 +146,7 @@ on web code only.
 is too strict, it's okay to `print` in some code.
 
 `avoid_returning_null`
-will be obsoleted by NNBD.
+will be obsoleted by sound null safety.
```
Comment (pq) [inline README.md:149]: 👍

File: CHANGELOG.md (hunk 1 above)
Comment (nshahan) [reply CHANGELOG.md:10]: Thanks, done!

File: CHANGELOG.md (hunk 2 above)
Comment (nshahan) [reply CHANGELOG.md]: Done.

File: README.md [hunk 4]
```diff
@@ -240,12 +252,15 @@ does not reflect common usage.
...
+users prefer `?:` over `if`/`else`.
+
 `prefer_int_literals`
 does not reflect common usage.
 
 `prefer_typing_uninitialized_variables`
-will be obsoleted by NNBD, which comes with type inference for uninitialized
-variables.
+will be obsoleted by sound null safety, which comes with type inference for
+uninitialized variables.
```
Comment (davidmorgan) [inline README.md:263]: This turned out to be wrong!... please remove from the banned list :)

**Review by davidmorgan — COMMENTED** [review]
Awesome, thanks :) one more change needed, then I'll merge + check + release.

Thanks!

File: README.md (hunk 4 above)
Comment (nshahan) [reply README.md]: Done.

**Comment by nshahan** [comment]
@davidmorgan I just realized I forgot to update the package version in the pubspec.yaml. Just want to leave you a note here that I pushed that change after your review.

**Comment by davidmorgan** [comment]
Awesome, thanks :) publishing...

**Pull request merged**